src/Restaurant.js

```diff
@@ -1,14 +1,14 @@
 import React from "react";
+import Order from "./Order";
 
 const Restaurant = () => {
-  const pizzas = 0;
   return (
-    <div>
+    <div className="restaurant-orders my-4">
       <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+      <ul className="list-group mx-2">
+        <Order orderType={"Pizzas"} />
+        <Order orderType={"Salads"} />
+        <Order orderType={"Chocolate cake"} />
       </ul>
```

Comment:
For lines 9 - 11, you don't need to use the curly braces since you are just passing in a hard-coded string.
You only need to use `{ }` if you are passing it some javascript.